_includes/cockroachcloud/ccloud/backup-details-dedicated.md

```diff
@@ -0,0 +1,8 @@
+- The date and time the backup was taken (**Data From**)
```

Comment:
I wasn't super clear here, I actually intended for the frequency of the backups to be in include files for each of the products. I don't think it is necessary at this point though. 

The content here does not need to be include files. So both:

_includes/cockroachcloud/ccloud/backup-details-dedicated.md
_includes/cockroachcloud/ccloud/backup-details-serverless.md

You can add the content back to the page as it was previously and delete these include files.